PrioritizedPartsList.csv

```diff
@@ -62,6 +62,7 @@ White Board Cleaning Cloth,$4.34 ,2,$8.68 ,http://amzn.to/2fuY4xA,9
 DC Power Supply,$79.99 ,2,$159.98 ,http://amzn.to/2x04v79,10
 Function Generator,$318.89 ,1,$318.89 ,http://amzn.to/2xBRS1v,10
 Digtial Oscilloscope,$379.00 ,2,$758.00 ,http://amzn.to/2fCwv9o,10
+Confetti Cannon, $1000000, 4, $4000000, https://confetticannon.ru
```

Comment:
This is the most ridiculous idea, I've ever heard. Seriously, you're insane. I like it.